dpdata/plugins/openmx.py

```
@@ -61,12 +61,12 @@ def from_labeled_system(self, file_name: str, **kwargs) -> dict:
         fname = f"{file_name}.dat"
         mdname = f"{file_name}.md"
 
-        data, cs = dpdata.openmx.omx.to_system_data(fname, mdname)
-        data["coords"] = dpdata.md.pbc.apply_pbc(
+        data, cs = dpdata.formats.openmx.omx.to_system_data(fname, mdname)
```

Ruff (0.15.12)

[warning] Unpacked variable `cs` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)